client/resources/languages/fr.json

```
@@ -5,48 +5,48 @@
   "login error": "Nom d‘utilisateur ou mot de passe incorrect",
 
   "suggestions": "Suggestions",
-  "suggestions for": "Suggestions for",
-  "refresh infos": "Refresh infos",
+  "suggestions for": "Suggestions pour",
+  "refresh infos": "Actualiser les infos",
   "delete item": "Supprimer",
   "confirm delete": "Es-tu sûr(e) ?",
   "search": "Rechercher",
-  "search for": "Search for",
-  "search or add": "Rechercher ou ajouter",
+  "search for": "Chercher",
+  "search or add": "Chercher ou ajouter",
   "load more": "Afficher plus",
   "nothing found": "Aucun résultat trouvé",
 
-  "calendar": "Calendar",
+  "calendar": "Calendrier",
   "trending": "Tendances",
   "upcoming": "Prochainement",
-  "now-playing": "Now Playing",
+  "now-playing": "En cours de lecture",
   "tv": "Séries TV",
-  "episodes": "Episodes",
+  "episodes": "Épisodes",
   "movies": "Films",
-  "movie": "Movie",
+  "movie": "Film",
   "last seen": "Dernier film/épisode vu",
-  "own rating": "Own Rating",
-  "title": "Title",
-  "release": "Release",
-  "tmdb rating": "TMDb Rating",
-  "imdb rating": "IMDb Rating",
+  "own rating": "Ma note",
+  "title": "Titre",
+  "release": "Sortie",
+  "tmdb rating": "Note TMDb",
+  "imdb rating": "Note IMDb",
   "change color": "Changer la couleur",
 
   "settings": "Préférences",
   "logout": "Se déconnecter",
   "tab user": "Utilisateur",
   "tab options": "Options",
-  "tab backup": "Backup",
+  "tab backup": "Sauvergarde",
   "tab misc": "Divers",
   
-  "finished": "Finished",
-  "episodes today": "Episodes today",
-  "movies today": "Movies today",
-  "daily reminder": "Daily",
-  "weekly reminder": "Weekly summary",
-  "reminders send to": "Send reminders to",
-  "reminders": "Reminders",
-  "refresh": "Refresh",
-  "refresh automatically": "Refresh items automatically",
+  "finished": "Terminé",
+  "episodes today": "Épisodes du jour",
+  "movies today": "Films du jour",
+  "daily reminder": "Quotidien",
+  "weekly reminder": "Résumé hebdomadaire",
+  "reminders send to": "Envoyer les rappels à",
+  "reminders": "Rappels",
+  "refresh": "Actualiser",
+  "refresh automatically": "Actualiser automatiquement",
```

Ce n'est déjà pas très explicite avec un terme verbeux alors je préférerais garder cette notions d'entrée.
```suggestion
  "refresh automatically": "Actualiser les informations automatiquement",
```